manifests/s/SKDataAS/ORMCore/202208262/SKDataAS.ORMCore.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created with YamlCreate.ps1 v2.1.4 $debug=QUSU.5-1-22000-832
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: SKDataAS.ORMCore
+PackageVersion: "202208262"
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: user
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT
```

Comment:
```suggestion
```
Not needed since `/verysilent` and `/silent` are defaults for `InstallerType: inno`